modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -371,10 +419,24 @@ protected function render() {
 				'aria-expanded' => 'false',
 			] );
 
+			$this->add_render_attribute( 'icon-selected', 'class', 'elementor-icon-selected' );
+			$this->add_render_attribute( 'icon-active', 'class', 'elementor-icon-active' );
+
 			$title_render_attributes = $this->get_render_attribute_string( $tab_title_setting_key );
 			$mobile_title_attributes = $this->get_render_attribute_string( $tab_title_mobile_setting_key );
-
-			$tabs_title_html .= "<div $title_render_attributes>$tab_title</div>";
+			$tab_icon_attributes = $this->get_render_attribute_string( 'icon-selected' );
+			$tab_icon_active_attributes = $this->get_render_attribute_string( 'icon-active' );
+
+			$icon_html = Icons_Manager::try_get_icon_html( $item['selected_icon'], [ 'aria-hidden' => 'true' ] );
+			$icon_active_html = $icon_html;
+			if ( $this->is_active_icon_exist( $item ) ) {
+				$icon_active_html = Icons_Manager::try_get_icon_html( $item['active_icon'], [ 'aria-hidden' => 'true' ] );
+			}
+			$tabs_title_html .= "<div $title_render_attributes>
+<span $tab_icon_attributes> $icon_html</span>
+<span $tab_icon_active_attributes> $icon_active_html</span>
+<span>$tab_title</span>
+</div>";
```

Comment:
Fix formating

Reply:
Here is example: https://github.com/elementor/elementor/blob/master/includes/widgets/accordion.php#L561